Source/Scene/Scene.js

```diff
@@ -1745,32 +1745,25 @@ define([
 
             // Don't insert globe commands with the rest of the scene commands since they are handled separately
             if (command.castShadows && (insertAll || (command.pass === Pass.OPAQUE || command.pass === Pass.TRANSLUCENT))) {
-                if (isPointLight) {
-                    if (defined(command.boundingVolume)) {
-                        var distance = command.boundingVolume.distanceSquaredTo(center);
-                        if (distance < radiusSquared) {
-                            for (var k = 0; k < numberOfPasses; ++k) {
-                                passes[k].commandList.push(command);
-                            }
+                if (isVisible(command, shadowVolume)) {
```

Comment:
Side note: once we get pre-culled globe commands from `Globe.update` we won't need to do this check for those.